File: 782-jewels-and-stones/jewels-and-stones.js
/**
 * @param {string} jewels
 * @param {string} stones
 * @return {number}
 */
var numJewelsInStones = function(jewels, stones) {
    let ans = 0;
    for(let i = 0; i < stones.length; i++){
        let currStone = stones[i];
        for(let j = 0; j < jewels.length; j++){
            if(currStone === jewels[j]){
                ans++;
                break;
            }
        }
    }
    return ans;
};

File: 782-jewels-and-stones/jewels-and-stones.java
class Solution {
    public int numJewelsInStones(String jewels, String stones) {
        int ans = 0;
        for (int i = 0; i < stones.length(); i++) {
            char currStone = stones.charAt(i);
            for (int j = 0; j < jewels.length(); j++) {
                if (currStone == jewels.charAt(j)) {
                    ans++;
                    break;
                }
            }
        }
        return ans;
    }
}